00-open-a-shapefile.Rmd

```diff
@@ -78,237 +75,254 @@ Working With Shapefile Attributes in R </a>
+library(raster)    #for metadata/attributes- vectors or rasters
+```
+
+The shapefiles we will import are:
+
+* a polygon shapefile representing our field site boundary, 
 * a line shapefile representing roads, and 
-* a point shapefile representing the location of a flux tower 
-<a href="http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements" target="_blank">
-NEON Flux Tower Measurement Methods</a> 
-located at the NEON Harvard Forest field site 
-<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
-NEON Harvard Forest Field Site</a>.
-
-The first shapefile that we will open contains the boundary of our study area. 
-We can query the layer type either by entering the name of the `R` object into the console 
-or by calling the `class()` function. In this case, we see that our object is a type of 
-polygon layer called a `SpatialPolygonsDataFrame`.
-
-Bonus:  The acronym, `OGR` refers to the OpenGIS Simple Features Reference 
-Implementation. 
-<a href="https://trac.osgeo.org/gdal/wiki/FAQGeneral" target="_blank"> Learn more about OGR.</a>
-{ : .notice }  
+* a point shapefile representing the location of a  
+<a href="http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements" target="_blank"> flux tower</a> 
+located at the
+<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank"> NEON Harvard Forest field site</a>.
 
-The first shapefile that we will open, contains the boundary of our study area
-(or our Area Of Interest, hence the name `aoiBoundary`). 
-It is a polygon layer. We can tell it is a polygon by viewing it's `class` in
-the attributes of the object which is `SpatialPolygonsDataFrame`.
+The first shapefile that we will open contains the boundary of our study area
+(or our Area Of Interest, hence the name `aoiBoundary`). To import shapefiles we
+use the `R` function `readOGR()`.
 
 ```{r Import-Shapefile}
 #set working directory to the directory location on your computer where
-#you downloaded and unzipped the data files used in this lesson
+#you downloaded and unzipped the data files for the lesson
 #setwd("pathToDirHere")
 
-#Import a polygon shapefile 
-aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
-
+#Import a polygon shapefile: readOGR("path","fileName")
+#no extension needed as readOGR only imports shapefiles
+aoiBoundary_HARV <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 
 ```
 
-After loading a new data set we also want to view the attributes of the layer.
-This can be done simple by entering the file name. 
-
-NOTE: we can also use the `attributes()` method to view objects attributes
-{: .notice }
-
-``` {r Shapefile-attributes}
-#view attributes of the layer
-aoiBoundary
-```
+Note: The acronym, `OGR` refers to the OpenGIS Simple Features Reference 
+Implementation. 
+<a href="https://trac.osgeo.org/gdal/wiki/FAQGeneral" target="_blank"> 
+Learn more about OGR.</a> {: .notice1}  
```

Comment:
**Data Tip** - (see template guide!)

{: .notice } - BELOW the paragraph this won't render properly.